package.json

```diff
@@ -52,6 +52,7 @@
     "react-dom": "18.2.0",
     "react-hook-form": "^7.49.3",
     "react-icons": "^5.0.1",
+    "react-loading-animation": "^1.4.0",
```

Comment:
This package was published 7 years ago and I don't think we need loading components as external dependency